core-services/prow/02_config/_config.yaml

```diff
@@ -58,6 +58,13 @@ branch-protection:
       repos:
         toolchain-operator:
           protect: false
+    integr8ly:
+      repos:
+        integreatly-operator:
+          exclude:
+          - master
+          - release-*
+          protect: false
```

Comment:
This will turn off branch protection for everything but master and our release branches right?

Reply:
Correct. That's based on the config for [operator-sdk repo](https://github.com/openshift/release/blob/a7e608c7393e4d907e5603d74f12e1515b457eff/core-services/prow/02_config/_config.yaml#L340-L344).

[This is the result](https://github.com/operator-framework/operator-sdk/branches)